PSMSVirtualDeviceManager/StartPage/UCStartPage.vb

```diff
@@ -521,7 +526,7 @@ Public Class UCStartPage
     Private Sub LinkLabel_ConfigToolClose_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles LinkLabel_ConfigToolClose.LinkClicked
         Try
             Dim mProcesses As Process() = Process.GetProcessesByName("PSMoveConfigTool")
-            If (mProcesses Is Nothing OrElse mProcesses.Length < 1) Then
+            If CheckIfConfigToolRunning() = False Then
```

Comment:
I prefer using ``Not`` rather than ``= False``. Also statements use parenthesis in this repo.

```suggestion
            If (Not CheckIfConfigToolRunning()) Then
```